common/src/java/com/zimbra/common/util/ZimbraCookie.java

```diff
@@ -27,6 +27,7 @@
     public static final String COOKIE_ZM_AUTH_TOKEN       = "ZM_AUTH_TOKEN";
     public static final String COOKIE_ZM_ADMIN_AUTH_TOKEN = "ZM_ADMIN_AUTH_TOKEN";
     public static final String COOKIE_ZM_TRUST_TOKEN      = "ZM_TRUST_TOKEN";
+    public static final String COOKIE_ZM_JWT      = "ZM_JWT";
```

Comment:
Just a cosmetic thing: You may want to align the "= ZM_JWT" to other declarations above.

Reply:
done